src/components/PerformanceOverlay.tsx

```
@@ -35,7 +35,9 @@ export function PerformanceOverlay({
   });
   
   const frameCountRef = useRef(0);
-  const lastTimeRef = useRef(performance.now());
+  // eslint-disable-next-line react-hooks/purity
+  // eslint-disable-next-line react-hooks/purity
+  const lastUpdateRef = useRef(performance.now());
   const lastUpdateRef = useRef(performance.now());
```

Biome (2.1.2)

[error] Shouldn't redeclare 'lastUpdateRef'. Consider to delete it or rename it.

'lastUpdateRef' is defined here:


(lint/suspicious/noRedeclare)